src/main/resources/com/cloudbees/jenkins/plugins/advisor/AdvisorGlobalConfiguration/index.jelly

```diff
@@ -4,134 +4,141 @@
   <l:layout title="${it.actionTitleText}" norefresh="true" permission="${it.ADMINISTRATOR}">
 
     <l:header>
-      <link rel="stylesheet" href="${resURL}/bootstrap/css/bootstrap.min.css" type="text/css"/>
       <link rel="stylesheet" href="${resURL}/css/font-awesome/css/font-awesome.min.css"/>
       <style type="text/css">
-        table {
-        border-collapse: separate;
-        border-spacing: 5px;
+        /* TODO remove when baseline newer than 2.245, https://github.com/jenkinsci/jenkins/pull/4842 */
+        .jenkins-health-advisor .alert-success {
+          background-color: #d4edda;
+          border-color: #c3e6cb;
+          color: #138347;
```

Comment:
This should work for theming, right @timja ?

```suggestion
          background-color: #d4edda;
          background-color: var(--alert-success-bg-color);
          border-color: #c3e6cb;
          border-color: var(--alert-success-border-color);
          color: #138347;
          color: var(--alert-success-text-color);
```

Reply:
yes perfect